frontend/src/lib/questions.ts

```diff
@@ -0,0 +1,77 @@
+import { Question, QuestionComplexity } from '@/types/question';
+
+export const getQuestions = () => {
+  const questions: Question[] = JSON.parse(
+    localStorage.getItem('questions') || '[]',
+  );
+  return questions;
+};
+
+export const createQuestion = (question: Question) => {
+  const questions: Question[] = getQuestions();
+  if (questions.length === 0) {
+    questions.push(question);
+    localStorage.setItem('questions', JSON.stringify(questions));
+    return;
+  }
+  // basic error handling to check for duplicates
+  const questionTitles: string[] = questions.map(
+    (qn: Question) => qn.questionTitle,
+  );
+  if (questionTitles.includes(question.questionTitle)) {
+    throw new Error('Question already exists');
+  }
+  questions.push(question);
+  localStorage.setItem('questions', JSON.stringify(questions));
+};
+
+// One-indexed id
+export const getQuestionById = (id: number) => {
+  const questions: Question[] = getQuestions();
+  return questions[id - 1];
+};
+
+// One-indexed id
+export const deleteQuestionById = (id: number) => {
+  const questions: Question[] = getQuestions();
+  questions.splice(id - 1, 1);
+  localStorage.setItem('questions', JSON.stringify(questions));
+};
+
+export const testing = () => {
+  localStorage.setItem('questions', JSON.stringify({}));
```

Comment:
Should this bug be fixed especially if we are using this code for testing? If we will not use this for future testing, should it be removed?